lib/beaker/host/windows.rb

```diff
@@ -24,5 +24,13 @@ def platform_defaults
       })
     end
 
+    attr_reader :network_separator, :external_copy_base
+    def initialize name, host_hash, options
+      super
+
+      @network_separator = '\\'
+      @external_copy_base = '`cygpath -smF 35`/'
```

Comment:
Also of note... I think that `@external_copy_base` should be resolvable at runtime like you're doing here.

However, if you look at how you've implemented in `host/pswindows.rb` and `host/unix.rb`, you'll see that `@external_copy_base` is set to a static value.  Can `opts[:copy_dir_external]` be resolved at runtime like this?